libpod/container.go

```diff
@@ -1336,3 +1336,29 @@ func (c *Container) getNetworkStatus() map[string]types.StatusBlock {
 	}
 	return nil
 }
+
+func (c *Container) UtsMode(ctrSpec *spec.Spec) spec.LinuxNamespaceType {
```

Comment:
Either return a sting or a constant. In this case you return `ns:....` which is not a valid Namespace type: https://github.com/containers/podman/blob/8e60baa1606379cc903f4dc0caccef040b5be62b/vendor/github.com/opencontainers/runtime-spec/specs-go/config.go#L201-L216

Reply:
Also looks like this is duplication which should be fixed in #14480

Reply:
I will wait for #14480 to merge @Luap99 so I can rebase and use that new function

Reply:
actually @Luap99 I need to keep this. The other function is a pod function this is for all containers (for inspect). I will make it the same syntax as the pod one in the mentioned PR